tests/devapps/XamarinDev/XamarinDev/App.xaml.cs

```diff
@@ -72,7 +72,7 @@ public static void InitPublicClient()
 
             if (UseBroker)
             {
-                //builder.WithBroker(true);
+                builder.WithBroker(true);
```

Comment:
>true [](start = 35, length = 4)

 Can't this simply be "empty" not sure where this dicussion landed previously

Reply:
@henrik-me yes...you're right. this is copy/paste from before those changes...thanks for pointing it out.